presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/ThriftConnectorStats.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.connector.thrift;
+
+import io.airlift.stats.DistributionStat;
+import org.weakref.jmx.Managed;
+import org.weakref.jmx.Nested;
+
+public class ThriftConnectorStats
+{
+    final DistributionStat pageSourcePageSize = new DistributionStat();
+    final DistributionStat pageSourcePageSizeNonEmpty = new DistributionStat();
```

Comment:
I think it's enough to get distribution on overall sizes. What I meant before was to calculate just the number of calls that return empty data. However, it doesn't seem necessary yet, so we can add it later.